api/types/registry/registry.go

```diff
@@ -102,3 +105,15 @@ type SearchResults struct {
 	// Results is a slice containing the actual results for the search
 	Results []SearchResult `json:"results"`
 }
+
+// DistributionInspect describes the result obtained from contacting the
+// registry to retrieve image metadata
+type DistributionInspect struct {
+	// Digest is the content addressable digest for the image on the registry
+	Digest digest.Digest
+	// ID is the local image digest
+	ID digest.Digest
```

Comment:
Based on this, I'm inclined to remove the `ID` field altogether, particularly because we aren't planning to use it right away. We can add it when we actually need it.